manual/src/ornate/reference.md

```diff
@@ -187,12 +187,6 @@ where you used to enable `WebScalaJS`).
 The plugin tunes the `scalaJSPipeline` to use the bundles produced by webpack rather than the direct
 output of the Scala.js compilation.
 
-> {.note}
-> The [scalajs-scripts](https://github.com/vmunier/scalajs-scripts) will **not** work anymore because they
-> try to include the output of Scala.js instead of the output of the bundling process. You can see
-> [here](https://github.com/scalacenter/scalajs-bundler/blob/master/sbt-web-scalajs-bundler/src/sbt-test/sbt-web-scalajs-bundler/play/server/src/main/scala/example/ExampleController.scala#L25-L30)
```

Comment:
Thanks for fixing the instructions. If you have time, it would be even better to update that test to use the `scalajs-scripts` instead of our _ad-hoc_ technique.

Reply:
Ok, I've pushed an update to the test, it appears to run fine locally for me.